### ISSUE TITLE
Install METAL

### ISSUE THREAD

**Issue by rmn8** [ubccr/software-layer#295]
Hi, I'd like to install METAL in UBCCR 
https://csg.sph.umich.edu/abecasis/Metal/download/

**Comment by tonykew** [comment]
Metal has been built, and is going through internal testing.
I will update this request once Metal has been published.

**Comment by tonykew** [comment]
Metal version 2020-05-05 has been published to the ccrsoft/2023.01 software release

```
$ module spider metal
----------------------------------------------------------------------------
  metal: metal/2020-05-05
----------------------------------------------------------------------------
    Description:
      Metal - Meta Analysis Helper. The METAL software is designed to
      facilitate meta-analysis of large datasets (such as several whole
      genome scans) in a convenient, rapid and memory efficient manner.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "metal/2020-05-05" module is available to load.

      gcc/11.2.0
[...]
$ module load gcc/11.2.0 metal/2020-05-05
$ metal
MetaAnalysis Helper - (c) 2007 - 2009 Goncalo Abecasis
This version released on 2020-05-05

# This program faciliates meta-analysis of genome-wide association studies.
# Commonly used commands are listed below:
#
# Options for describing input files ...
#   SEPARATOR        [WHITESPACE|COMMA|BOTH|TAB] (default = WHITESPACE)
#   COLUMNCOUNTING   [STRICT|LENIENT]            (default = 'STRICT')
#   MARKERLABEL      [LABEL]                     (default = 'MARKER')
#   ALLELELABELS     [LABEL1 LABEL2]             (default = 'ALLELE1','ALLELE2')
#   EFFECTLABEL      [LABEL|log(LABEL)]          (default = 'EFFECT')
#   FLIP
#
# Options for filtering input files ...
#   ADDFILTER        [LABEL CONDITION VALUE]     (example = ADDFILTER N > 10)
#                    (available conditions are <, >, <=, >=, =, !=, IN)
#   REMOVEFILTERS
#
# Options for sample size weighted meta-analysis ...
#   WEIGHTLABEL      [LABEL]                     (default = 'N')
#   PVALUELABEL      [LABEL]                     (default = 'PVALUE')
#   DEFAULTWEIGHT    [NUMBER]                    (default = 1.0)
#   MINWEIGHT        [NUMBER]                    (default = 1.0)
#
# Options for inverse variance weighted meta-analysis ...
#   STDERRLABEL      [LABEL]                     (default = 'STDERR')
#   SCHEME           [SAMPLESIZE|STDERR]         (default = SAMPLESIZE)
#
# Options to enable tracking of allele frequencies ...
#   AVERAGEFREQ      [ON|OFF]                    (default = OFF)
#   MINMAXFREQ       [ON|OFF]                    (default = OFF)
#   FREQLABEL        [LABEL]                     (default = 'FREQ')
#
# Options to enable tracking of user defined variables ...
#   CUSTOMVARIABLE   [VARNAME]
#   LABEL            [VARNAME] AS [HEADER]
#
# Options to enable tracking of chromosomes and positions ...
#   TRACKPOSITIONS   [ON|OFF]                    (default = OFF
#   CHROMOSOMELABEL  [LABEL]                     (default = 'CHROMOSOME')
#   POSITIONLABEL    [LABEL]                     (default = 'POSITION')
#
# Options to enable explicit strand information ...
#   USESTRAND        [ON|OFF]                    (default = OFF)
#   STRANDLABEL      [LABEL]                     (default = 'STRAND')
#
# Automatic genomic control correction of input statistics ...
#   GENOMICCONTROL   [ON|OFF|VALUE|LIST snps.txt](default = OFF)
#
# Options to account for samples overlap ...
#   OVERLAP          [ON|OFF]                    (default = OFF)
#   ZCUTOFF          [NUMBER]                    (default = 1.0)
#
# Options for general analysis control ...
#   PROCESSFILE            [FILENAME]
#   OUTFILE                [PREFIX SUFFIX]       (default = 'METAANALYSIS','.TBL')
#   MAXWARNINGS            [NUMBER]              (default = 20)
#   VERBOSE                [ON|OFF]              (default = 'OFF')
#   LOGPVALUE              [ON|OFF]              (default = 'OFF')
#   EFFECT_PRINT_PRECISION [NUMBER]              (default = '4')
#   STDERR_PRINT_PRECISION [NUMBER]              (default = '4')
#   ANALYZE                [HETEROGENEITY]
#   CLEAR

# Options for general run control ...
#   SOURCE           [SCRIPTFILE]
#   RETURN
#   QUIT

QUIT
## Clearing all stored statistics ...
# Clearing user defined filters ...
$ 

```
